src/ShellProgressBar/target.js

```diff
@@ -0,0 +1,122 @@
+(function ($R) {
```

Comment:
Thanks for the PR! Can you remove this file? 

Curious how did this file make it in here?